src/core/runtime.hpp

```diff
@@ -382,6 +378,44 @@ struct SHMesh : public std::enable_shared_from_this<SHMesh> {
 
   ~SHMesh() { terminate(); }
 
+  void compose(const std::shared_ptr<SHWire> &wire, SHVar input = shards::Var::Empty) {
```

Comment:
Check comment in SHCore malMesh as well.
This code seems very similar to that in SHMesh::schedule, can we remove the compose code from schedule and instead have callers call:
```
compose(...)
schedule(...)
```
separately?

Reply:
yeah but what about edn code? keep `schedule` abstraction right?

Reply:
Leaving this out as requires a bigger refactor that is a waste of time for me sadly @guusw